src/components/common/SideNavigation/index.tsx

```diff
@@ -14,7 +15,7 @@ export default function SideNavigation(): JSX.Element {
   const auth = useAuth();
   const clearAuth = useClearAuth();
   const location = useLocation();
-  const [visible, , , toggle, setValue] = useBooleanState(false);
+  const [visible, , , toggle, setValue] = useBooleanState(true);
```

Comment:
마이너) setValue보단 적절하게 visible로 통일해도 좋을 것 같아요
```suggestion
  const [visible, , , toggle, setVisible] = useBooleanState(true);
```

Reply:
수정했습니다. [173232d](https://github.com/BCSDLab/JJBAKSA_FRONT_END/pull/107/commits/173232d5fff93e1bed2ba0e39f5f37bf80144e47)